crates/lib/src/qpu/api.rs

```diff
@@ -131,9 +131,8 @@ pub async fn submit(
 /// This action is *not* atomic, and will attempt to cancel every job even when some jobs cannot be
 /// cancelled. A job can be cancelled only if it has not yet started executing.
 ///
-/// Success response indicates only that the request was received. Cancellation is not guaranteed,
-/// as it is based on job state at the time of cancellation, and is completed on a best effort
-/// basis.
+/// Cancellation is not guaranteed, as it is based on job state at the time of cancellation, and is
+/// completed on a best effort basis.
```

Comment:
Should be explicit about whether an error is returned if a job cannot be cancelled. As a user, I would also want to know what would happen if I passed in two jobs, where one is already executing but the other can be cancelled.

Reply:
FWIW the idiomatic thing to do is have a
```rust
/// # Errors
```
section in the comment

Reply:
May also replace some of this comment with a "See `cancel_job`"

Reply:
Added a note on what errors to expect when job cancellation fails.